ui/dashboards/src/validation/panel.ts

```diff
@@ -17,7 +17,10 @@ export const panelEditorValidationSchema = z.object({
   name: z.string().nonempty('Required'),
   groupId: z.number(),
   description: z.string().optional(),
-  type: z.string().nonempty('Required'),
+  selection: z.object({
+    type: z.string(),
```

Comment:
Having both kind and type is confusing, I thought it was `kind: ‘Panel’`? Is `type` going in the spec too or is this is something different?

Reply:
answered there https://github.com/perses/perses/pull/1899#discussion_r1560413645